File: remote/src/main/java/com/kelsos/mbrc/events/ui/TrackRemoval.java
package com.kelsos.mbrc.events.ui;

import org.codehaus.jackson.node.ObjectNode;

public class TrackRemoval {
  private int index;
  private boolean success;

  public TrackRemoval(ObjectNode node) {
    index = node.path("index").asInt();
    success = node.path("success").asBoolean();
  }

  public int getIndex() {
    return this.index;
  }

  public boolean isSuccess() {
    return success;
  }
}

File: app/src/main/java/com/kelsos/mbrc/events/ui/TrackRemoval.kt
package com.kelsos.mbrc.events.ui

import com.fasterxml.jackson.databind.node.ObjectNode

class TrackRemoval(
  node: ObjectNode,
) {
  val index: Int = node.path("index").asInt()
  val isSuccess: Boolean = node.path("success").asBoolean()
}
